### PR TITLE
TS-2

### PR DESCRIPTION
**Pull request by kav128**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+### Java ###
+# Compiled class file
+*.class
+
+# Log file
+*.log
+
+# BlueJ files
+*.ctxt
```
Comment (VladislavChernykh) [inline .gitignore]: Please remove the extensions are not in use.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
```
Comment (VladislavChernykh) [inline .gitignore]: Same as above

File: .gitignore (hunk 1 above)
Comment (kav128) [reply .gitignore]: Removed some lines and added Maven's `target/`

**Review by VladislavChernykh — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (VladislavChernykh) [reply .gitignore]: It might be useful to keep .log files excluded from the repository (in case you will use any logging system in the project)

**Pull request merged**